.rubocop.yml

```diff
@@ -54,3 +54,7 @@ Style/AndOr:
 
 Style/DoubleNegation:
   Enabled: false
+
+Style/DateTime:
+  Exclude:
+    - 'spec/lib/active_data/model/typecasting_spec.rb'
```

Comment:
Why don't leave the comment in the file instead?

Reply:
Fixed it :slightly_smiling_face: